libbeat/outputs/fileout/file.go

```diff
@@ -19,8 +19,11 @@ package fileout
 
 import (
 	"context"
+	"github.com/elastic/go-ucfg"
```

Comment:
Could you run `mage check` in the `libbeat` folder? That will fix this lint issue and allow the CI check step to pass.

Among other  things `mage check` will re-format all files following our policy which will fix the import order.

Reply:
done!